### PR TITLE
ResNet v1d

### PR DESCRIPTION
**Pull request by Jerryzcn**
add avgpool to projection connection in resnet.
add dropout

78.4% top-1 acc on imagenet val

### CODE REVIEW

**Comment by mli** [comment]
Job PR-232-2 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-232/2/index.html 
Code coverage of this PR:  vs. Master:

File: gluoncv/model_zoo/resnetv1b.py [hunk 1]
```diff
@@ -445,3 +464,73 @@ def resnet152_v1c(pretrained=False, root='~/.mxnet/models', ctx=cpu(0), **kwargs
...
+    root : str, default '~/.mxnet/models'
+        Location for keeping the model parameters.
+    ctx : Context, default CPU
+        The context in which to load the pretrained weights.
+    dilated: bool, default False
+        Whether to apply dilation strategy to ResNetV1b, yilding a stride 8 model.
+    norm_layer : object
+        Normalization layer used in backbone network (default: :class:`mxnet.gluon.norm_layer`;
+    """
+    model = ResNetV1b(BottleneckV1b, [3, 4, 6, 3], deep_base=True, avg_down=True, final_drop=0.2,
```
Comment (hetong007) [inline gluoncv/model_zoo/resnetv1b.py]: Please include new parameters in the docstring for 50, 101, and 152.

**Comment by mli** [comment]
Job PR-232-4 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-232/4/index.html 
Code coverage of this PR:  vs. Master:

File: gluoncv/model_zoo/resnetv1b.py [hunk 2]
```diff
@@ -439,9 +464,78 @@ def resnet152_v1c(pretrained=False, root='~/.mxnet/models', ctx=cpu(0), **kwargs
...
+    root : str, default '~/.mxnet/models'
+        Location for keeping the model parameters.
+    ctx : Context, default CPU
+        The context in which to load the pretrained weights.
+    dilated: bool, default False
+        Whether to apply dilation strategy to ResNetV1b, yilding a stride 8 model.
+    norm_layer : object
+        Normalization layer used in backbone network (default: :class:`mxnet.gluon.norm_layer`;
+    """
+    model = ResNetV1b(BottleneckV1b, [3, 4, 6, 3], deep_base=True, avg_down=True, **kwargs)
```
Comment (hetong007) [inline gluoncv/model_zoo/resnetv1b.py]: why `deep_base` here?

**Comment by mli** [comment]
Job PR-232-5 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-232/5/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-232-7 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-232/7/index.html 
Code coverage of this PR:  vs. Master:

**Comment by mli** [comment]
Job PR-232-8 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-232/8/index.html 
Code coverage of this PR:  vs. Master:

**Comment by hetong007** [comment]
Please add tests for new models at https://github.com/dmlc/gluon-cv/blob/master/tests/unittests/test_model_zoo.py#L87

**Comment by hetong007** [comment]
@Jerryzcn Lint error on CI is fixed. Can you please rebase with master?

**Comment by mli** [comment]
Job PR-232-12 is done. 
Docs are uploaded to http://gluon-vision-staging.s3-website-us-west-2.amazonaws.com/PR-232/12/index.html 
Code coverage of this PR:  vs. Master:

**Pull request merged**